Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -965,7 +873,8 @@ static u32 TranslateBlockAddress(const u32 addr, const XCheckTLBFlag _Flag)
 }
 
 // Translate effective address using BAT or PAT.  Returns 0 if the address cannot be translated.
-u32 TranslateAddress(const u32 _Address, const XCheckTLBFlag _Flag)
+template <const XCheckTLBFlag _Flag>
```

Comment:
Something like this:

```
template u32 TranslateAddress<Memory::FLAG_NO_EXCEPTION>(const u32 _Address);
template u32 TranslateAddress<Memory::FLAG_READ_EXCEPTION>(const u32 _Address);
template u32 TranslateAddress<Memory::FLAG_WRITE_EXCEPTION>(const u32 _Address);
```

This forces the compiler to generate the code even if there aren't any callers in the same file.